client/src/components/Lists/PitchTeams/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, ReactElement } from 'react';
+import { IPitch, IUser } from 'ssw-common';
+
+import { FieldTag } from '../..';
+import { useTeams } from '../../../contexts';
+
+interface PitchTeamsProps {
+  pitch: IPitch;
+  user?: Pick<IUser, '_id'>;
+  assignmentContributors?: boolean;
+}
+
+const PitchTeams: FC<PitchTeamsProps> = ({
+  pitch,
+  user,
+  assignmentContributors = false,
+}): ReactElement => {
+  const { getTeamFromId } = useTeams();
+
+  const getPitchTeams = (): string[] => {
+    if (user) {
+      let contributor: { teams: string[]; [x: string]: any } | undefined;
+      if (assignmentContributors) {
+        contributor = pitch.assignmentContributors.find(
+          (contributor) => contributor.userId === user._id,
+        );
+      } else {
+        contributor = pitch.pendingContributors.find(
+          (contributor) => contributor.userId === user._id,
+        );
+      }
+      return contributor?.teams ?? [];
```

Comment:
I realize now that this is better than using `||`. So many things I learn that I wish I could go back and fix everywhere LOL